Tasks/AzureStaticWebAppV0/index.ts

```diff
@@ -19,39 +32,127 @@ async function run() {
 
         bash.line(tl.getInput('args', false));
 
-        const deploymentClient = "mcr.microsoft.com/appsvc/staticappsclient:stable";
-
-        const workingDirectory: string = tl.getInput('cwd', false) || "";
-        const appLocation: string = tl.getInput('app_location', false) || "";
-        const appBuildCommand: string = tl.getInput('app_build_command', false) || "";
-        const outputLoction: string = tl.getInput('output_location', false) || "";
-        const apiLocation: string = tl.getInput('api_location', false) || "";
-        const apiBuildCommand: string = tl.getInput('api_build_command', false) || "";
-        const routesLocation: string = tl.getInput('routes_location', false) || "";
-        const skipAppBuild: boolean = tl.getBoolInput('skip_app_build', false);
-        const apiToken: string = process.env['azure_static_web_apps_api_token'] || tl.getInput('azure_static_web_apps_api_token', false) || "";
-
-        process.env['SWA_WORKING_DIR'] = workingDirectory;
-        process.env['SWA_APP_LOCATION'] = appLocation;
-        process.env['SWA_APP_BUILD_COMMAND'] = appBuildCommand;
-        process.env['SWA_OUTPUT_LOCATION'] = outputLoction;
-        process.env['SWA_API_LOCATION'] = apiLocation;
-        process.env['SWA_API_BUILD_COMMAND'] = apiBuildCommand;
-        process.env['SWA_ROUTES_LOCATION'] = routesLocation;
-        process.env['SWA_DEPLOYMENT_CLIENT'] = deploymentClient;
-        process.env['SWA_SKIP_APP_BUILD'] = skipAppBuild.toString();
-        process.env['SWA_API_TOKEN'] = apiToken;
-
+        await createDockerEnvVarFile(envVarFilePath);
+        
         const options = {
             failOnStdErr: false
         };
 
         await bash.exec(<any>options);
         tl.setResult(tl.TaskResult.Succeeded, null);
-    }
-    catch (err) {
+    } catch (err) {
         tl.setResult(tl.TaskResult.Failed, null);
+    } finally {
+        await fs.promises.unlink(envVarFilePath);
```

Comment:
```suggestion
        await fs.promises.unlink(envVarFilePath).catch(() => tl.debug("Unable to delete env file"));
```

If unable to delete the file, probably just need to log it (currently it throws an uncaught exception and fails the task).